docs/00_install/01_build-from-source/index.md

```diff
@@ -4,6 +4,10 @@ content_title: Build EOSIO from Source
 
 The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the README.md file. Instructions are also included for efficiently running the tests.
 
+### Using DUNE
+
+As an alternative to building from source try [Docker Utilities for Node Execution](https://github.com/eosnetworkfoundation/DUNE) for the easiest way to get started, and for multi-platform support.  
```

Comment:
This is quite a bit of a shift from the current messaging which is to encourage use of the binary builds. My understanding was that we were positioning DUNE for multi platform guided contract development, not for, say, running a long term node connected to a public network or setting up a local wallet for use on a public network. Indeed, DUNE runs too slow on something like a M1 to be usable on WAX etc due to the way mandel & DUNE are designed: we do not have the same level of support in DUNE as the binaries do.

Since this documentation is agnostic to the reason a user is setting up a node, I think we need to be more judicious in when we encourage DUNE.